SmartDeviceLink/SDLManager.m

```diff
@@ -119,12 +119,16 @@ - (nullable SDLProxy *)proxy {
 
 #pragma mark SDLConnectionManager Protocol
 
+- (void)sendRPC:(__kindof SDLRPCMessage *)rpc {
+    [self.lifecycleManager sendRequest:rpc];
```

Comment:
I used `sendRPC` because that is what they are already using in Android. I don't think "send message" is more or less confusing than "send RPC". 
Not sure if we should deprecate `sendRequest:` since `sendRPC:` can also send requests.